javascript/README.md

```diff
@@ -643,3 +651,101 @@ const { left, top } = processInput(input);
 ```
 
 [⇧ top](#javascript-guide)
+
+---
+
+## Strings
+
+### 5.1 Quotes
+
+Use single quotes `''` for strings.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+const name = "Cloud Four";
+
+// Template literals should contain interpolation or newlines.
+const name = `Cloud Four`;
+```
+
+🎉 Yep! 🎉
+
+```js
+const name = 'Cloud Four';
+```
+
+#### Resources
+
+- ESLint: [quotes]
+
+### 5.2 Template Literals
+
+When programmatically building up strings, use [template literals][Template Literals] instead of concatenation.
+
+> Why? Template literals (template strings) give you a readable, concise syntax with proper newlines and string interpolation features.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+function sayHi(name) {
+  return 'How are you, ' + name + '?';
+}
+
+function sayHi(name) {
+  return ['How are you, ', name, '?'].join();
+}
+```
+
+🎉 Yep! 🎉
+
+```js
+function sayHi(name) {
+  return `How are you, ${name}?`;
+}
+```
+
+#### Resources
+
+- ESLint:
+  - [prefer-template]
+  - [template-curly-spacing]
+
+### 5.3 Eval
+
+[Never use `eval()`][No eval] on a string, it opens too many vulnerabilities.
+
+#### Resources
+
+- ESLint: [no-eval]
+
+### 5.4 Escaping Characters
+
+Do not unnecessarily escape characters in strings.
```

Comment:
[Prettier automatically removes unnecessary escape characters](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAzCFMC8mA5ADqnEwAWAlmucZuTeWk+SAI4CuE8AJuVIhiAbhAAaEBAAOMGumSgAhgCdVEAO4AFNQjTIQygG4Qa-SSABGq5WADWcGAGUZdmlADmyGKu5wpDzQ4VRhtW08AW2VkbGUAG2CpACs0AA8AIVsHJ2dlSLgAGQ84WISkkDdVYNVDK2UrAE946EsZVQ8YAHVzamQADgAGKXaIYK7bGUN2uBrjUqlVOB4aJfDlKJikOMSAkGDImh8-PbQPT3i4AEVeeDLdqRgGnv4+pAAmR9saePOAYQgkWihig0AWIG4wQAKg0DNtynAAL6IoA). I don't know if this section is necessary